npm/webpack-dev-server-fresh/cypress/e2e/vue-cli.cy.ts

```diff
@@ -0,0 +1,31 @@
+// <reference types="cypress" />
+/// <reference path="../support/e2e.ts" />
+import type { e2eProjectDirs } from '@packages/frontend-shared/cypress/e2e/support/e2eProjectDirs'
+
+type ProjectDirs = typeof e2eProjectDirs
+
+const PROJECTS: ProjectDirs[number][] = ['vuecli4-vue2', 'vuecli5-vue3']
+
+// Add to this list to focus on a particular permutation
+// TODO: run vuecli4-vue2 tests once cypress/vue-2 is bundled
+const ONLY_PROJECTS: ProjectDirs[number][] = ['vuecli5-vue3']
```

Comment:
I think we should have a test for `vuecli4-vue3` as well.

Reply:
https://github.com/cypress-io/cypress/pull/20956/commits/384aa53fcdb0bba122c155efa69316c702728c97 and https://github.com/cypress-io/cypress/pull/20956/commits/602ca2001d38858e5fa06710d80d2b71b6372297